README.md

```diff
@@ -2,6 +2,14 @@
 
 [![](https://godoc.org/github.com/CyCoreSystems/audiosocket?status.svg)](http://godoc.org/github.com/CyCoreSystems/audiosocket)
 
+### --计划在原AudioSocket基础上，增加 DTMF 信号的传递功能，并不会实际产生和检测DTMF信息，需要在SOCKET另一端实现此功能。
```

Comment:
What are these extra characters?  They just show up as placeholder glyphs for me.